va/Makefile.am

```diff
@@ -103,6 +103,10 @@ libva_x11_la_LDFLAGS		= $(LDADD)
 libva_x11_la_DEPENDENCIES	= libva.la x11/libva_x11.la
 libva_x11_la_LIBADD		= libva.la x11/libva_x11.la \
 	$(LIBVA_LIBS) $(X11_LIBS) $(XEXT_LIBS) $(XFIXES_LIBS) $(DRM_LIBS) -ldl
+if USE_DRI3
+libva_x11_la_LIBADD		+= $(X11_XCB_LIBS) $(XCB_LIBS) \
+	$(XCB_DRI3_LIBS) -ldl
```

Comment:
These variables aren't mentioned anywhere else - I think you want DRI3_LIBS here.